src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as VjsLib from './lib';
```

Comment:
I'm regularly typing `import Lib from './lib'` and having to fix it. Can you remind me why this is better than just exporting a Lib object with the individual functions on it?